docs/topics/db/managers.txt

```diff
@@ -435,99 +425,3 @@ However, if you're overriding ``__getattr__`` or some other private
 method of your ``Manager`` object that controls object state, you
 should ensure that you don't affect the ability of your ``Manager`` to
 be copied.
-
-.. _manager-types:
-
-Controlling automatic manager types
-===================================
-
-This document has already mentioned a couple of places where Django creates a
-manager class for you: `default managers`_ and the "plain" manager used to
-`access related objects`_. There are other places in the implementation of
-Django where temporary plain managers are needed. Those automatically created
-managers will normally be instances of the :class:`django.db.models.Manager`
-class.
```

Comment:
**Notes**:

Now that we introduce APIs such as `Meta.default_manager_name`, `Meta.base_manager_name`, `Model._default_manager`, and `Model._base_manager`, we should stop referring to them as just concepts like "plain", or "automatic" but instead point to their respective documentation.

Regarding `Model._default_manager` and `Model._base_manager` we should make it explicit that these are the managers you should use any time you deal with undetermined models (e.g. Django generic views, or admin views), it's especially needed for 3rd party apps.

When documenting `_base_manager`, we should be more explicit about [actual usage in Django](https://github.com/django/django/pull/6175#issuecomment-187736454 lists every usage): cascading deletions, dumpdata, to-one relations). We probably want to mention that unless you know that you really want to use `_base_manager` you should use `_default_manager`.